packages/fetchai/skills/generic_buyer/tests/test_models.py

```diff
@@ -41,40 +41,39 @@ class TestGenericStrategy(BaseSkillTestCase):
 
     path_to_skill = PACKAGE_ROOT
 
-    @classmethod
-    def setup_class(cls):
```

Comment:
Why are you changing it? What was the problem with it being on the class level?

Reply:
the strategy not being reset. I've changed it now to introduce a teardown where only the strategy gets its original content reset